code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -5,11 +5,11 @@
 	var/active_w_class
 	sharp = 0
 	edge = 0
-	armor_penetration = 10
+	armor_penetration = 80
 	flags = NOBLOODY
 	can_embed = 0//No embedding pls
 	var/base_reflectchance = 40
-	var/base_block_chance = 25
+	var/base_block_chance = 40

```

Comment:
This has to be logged in changelog and PR description.

Reply:
I forgot I changed that! My bad.

Reply:
You still haven't reverted it or logged it. Do either